src/fez/cli.cr

```diff
@@ -1,9 +1,7 @@
 module Fez
   struct CLI
-    property application_name, application_directory
-
-    def self.run
-      cli = self.new
+    def self.run(options = ARGV)
+      cli = self.new(options.first? || "my_project")
```

Comment:
In one of my first iterations of fez, I had a default name, but ended up going away from that. I think the project name should be required.